axiom/nr_errors.h

```diff
@@ -54,6 +86,35 @@ extern const char* nr_error_get_message(const nr_error_t* error);
  * Returns : The klass of the error or NULL on failure.
  */
 extern const char* nr_error_get_klass(const nr_error_t* error);
+
+/*
+ * Purpose : Get the error file of an error.
+ *
+ * Returns : The error file of the error or NULL on failure.
```

Comment:
Is it a 'failure' or simply it is not defined?

Reply:
Good point, it should instead say that it was not defined, fixed: [9fad7fa](https://github.com/newrelic/newrelic-php-agent/pull/942/commits/9fad7fae54e3fa1a4a2d2dfecf7493e61ee1f2a3)